src/core/action.ts

```diff
@@ -2,6 +2,19 @@ import { ActionDescriptor, parseActionDescriptorString, stringifyEventTarget } f
 import { Token } from "../mutation-observers"
 import { camelize } from "./string_helpers"
 
+const keyMappings: { [key: string]: string } = {
```

Comment:
Could this possibility be put on the Stimulus application? Maybe this way the mapping can be modified on a per application basis this way.

Reply:
Like action_event?
https://github.com/hotwired/stimulus/blob/main/src/core/action_event.ts

Reply:
Sorry, the comment was not clear.

The use case would be running Stimulus but being able to pass in a custom (or extended) `keyMappings` object when initialising the application (maybe as part of the schema) https://github.com/hotwired/stimulus/blob/main/src/core/schema.ts

```
// src/application.js
import { Application, defaultSchema } from "@hotwired/stimulus"

const customSchema = {
  ...defaultSchema,
  keyMappings: {...defaultSchema.keyMappings, a: "a", w: "w", s: "s", d: "d" },
}

window.Stimulus = Application.start(document.documentElement, customSchema);
```

Reply:
Thank you for explaining it all so well! :heart:
I agree with your suggestion.
Give me a minute. :)